src/components/radioButton/RadioButton.js

```diff
@@ -165,21 +191,45 @@ class RadioButton extends BaseComponent {
     return style;
   }
 
+  getIconStyle() {
```

Comment:
I don't think there's a need to extract the functionality of getting the icon style into a separate method. 
Basically, this code can be part of `renderIcon`